include/z64scene.h

```diff
@@ -462,6 +462,7 @@ typedef struct {
 } MeshHeader2; // size = 0xC 
 
 #define PATH_INDEX_NONE -1
+#define ADDITIONAL_PATH_INDEX_NONE (u8)PATH_INDEX_NONE
```

Comment:
are we really tying every single "_PATH_INDEX_NONE" to a reference to `PATH_INDEX_NONE`

Reply:
I'll admit maybe I have been a bit carried away with correlating them all with `PATH_INDEX_NONE`. However I do think it is still important/a good idea to show these are all just different `-1`s expressed in different widths. With that in mind, I suggest change them all to be literals, but instead of expressing them as hex literals, `0x1F`, use `-1` with ands/casts, `((-1) & 0x1F)`. More or less this would just replace the `PATH_INDEX_NONE`s with the actual literal `-1`.

Reply:
I did eventually just decouple everything and just used the literal hex value i.e (0x3F).